evidence/DEP-SDG-005-MISSION-V5-ACTIVATION/rollback.md

```
@@ -0,0 +1,33 @@
+## Trigger
+
+Any authority expansion, unexpected path/action/mode, protected-review or
+hosted gate failure, non-exact merge topology, post-merge readback failure, or
+attempt to start T-004 before ACTIVE validation.
+
+## Reversible steps
+
+rollback_version: 1.0
+target: open SDG-005 Pull Request from agent/mission-v5-activation-post-sdg008 in zycaskevin/Vault-Agent-Memory before activation merge
+command: test "$(gh pr view agent/mission-v5-activation-post-sdg008 --repo zycaskevin/Vault-Agent-Memory --json state,headRefName,mergeCommit --jq 'select(.state == "OPEN" and .headRefName == "agent/mission-v5-activation-post-sdg008" and .mergeCommit == null) | .headRefName')" = "agent/mission-v5-activation-post-sdg008" && gh pr close agent/mission-v5-activation-post-sdg008 --repo zycaskevin/Vault-Agent-Memory
+verify: test "$(gh pr view agent/mission-v5-activation-post-sdg008 --repo zycaskevin/Vault-Agent-Memory --json state,headRefName,mergeCommit --jq 'select(.state == "CLOSED" and .headRefName == "agent/mission-v5-activation-post-sdg008" and .mergeCommit == null) | .headRefName')" = "agent/mission-v5-activation-post-sdg008" && verify_root="$(mktemp -d /tmp/vault-sdg005-rollback-XXXXXX)" && trap 'rm -rf "$verify_root"' EXIT && git clone --no-local --quiet . "$verify_root/repo" && git -C "$verify_root/repo" remote set-url origin https://github.com/zycaskevin/Vault-Agent-Memory.git && git -C "$verify_root/repo" -c advice.detachedHead=false checkout --detach 46690372e532c50761f9232ff5b2e20e18779d28 && test -z "$(git -C "$verify_root/repo" status --porcelain)" && (cd "$verify_root/repo" && python3 -c 'import json,pathlib,subprocess; p=subprocess.run(["python3","scripts/validate_subject_task_authorization_dispatch_v5.py","--ledger","--json"],capture_output=True,text=True); assert p.returncode == 0 and p.stderr == ""; value=json.loads(p.stdout); assert value == {"active":False,"mission_id":None,"mission_state":"INACTIVE","protocol_version":5,"sequence":6,"status":"PASS"}; ledger=json.loads(pathlib.Path("specs/subject-distillation/implementation-progress.json").read_text()); assert len(ledger["events"]) == 6 and ledger["tasks"]["T-004"] == "PENDING"; assert all(not pathlib.Path(item).exists() for item in ("specs/subject-distillation/.task-authorization.pending","specs/subject-distillation/.implementation-progress.pending","specs/subject-distillation/.development-mission-v5-revocation.pending"))')
```

markdownlint-cli2 (0.23.2)

[warning] Reference links and images should use a label that is defined
Missing link or image reference definition: ""t-004""

(MD052, reference-links-images)